### PR TITLE
update jck.mk with subtest groups

### PR DESCRIPTION
**Pull request by KapilPowar**
[#1181](https://github.ibm.com/runtimes/backlog/issues/1181)

Signed-off-by: Kapil Powar [kapil.powar@in.ibm.com](mailto:kapil.powar@in.ibm.com)

CC: @llxia

### CODE REVIEW

**Comment by llxia** [comment]
Covert to draft until it is verified in Grinder

File: jck/jck.mk [hunk 1]
```diff
@@ -569,73 +569,73 @@ ifeq (21, $(JDK_VERSION))
...
+   COMPILER_LANG_CLASS_TESTS_GROUP1=$(Q)lang/CLSS/clss001;lang/CLSS/clss004;lang/CLSS/clss006;lang/CLSS/clss008;lang/CLSS/clss009;lang/CLSS/clss010;lang/CLSS/clss012;lang/CLSS/clss015;lang/CLSS/clss016;lang/CLSS/clss018;lang/CLSS/clss019;lang/CLSS/clss020;lang/CLSS/clss021;lang/CLSS/clss022;lang/CLSS/clss023;lang/CLSS/clss024;lang/CLSS/clss025;lang/CLSS/clss026;lang/CLSS/clss027;lang/CLSS/clss028;lang/CLSS/clss029;lang/CLSS/clss030;lang/CLSS/clss031;lang/CLSS/clss033;lang/CLSS/clss034;lang/CLSS/clss035;lang/CLSS/clss038;lang/CLSS/clss044;lang/CLSS/clss045$(Q)
```
Comment (llxia) [inline jck/jck.mk:572]: `lang/CLSS/clss002;lang/CLSS/clss003;` are we going to remove them?

Comment (KapilPowar) [reply jck/jck.mk]: re run splitter.sh against compiler-21 folder and updated the file.

**Comment by llxia** [comment]
FYI @JasonFengJ9 . For details, please see backlog/issues/1181

File: jck/jck_target_folder_check/README.md [hunk 2]
```diff
@@ -10,19 +10,22 @@ The jck_target_folder_check.sh script is used to cross-check if all second-level
...
+```
+$ jck_target_folder_check.sh git@github.ibm.com:runtimes/JCK21-unzipped.git <LOCAL_AQA_TESTS_REPO>
 ```
 
 # What the script does 
 
 
 ```
-1. Git clone the the given JCK repo (if they do not already exist under given {somerWorkspaceLocation}. 
+1. Git clone the the given JCK repo (if they do not already exist under given {somerWorkspaceLocation}).
```
Comment (renfeiw) [inline jck/jck_target_folder_check/README.md:28]: Should be `{someWorkspaceLocation}`? I think `{workspaceLocation}` will be more professional. Please also update the other places as well.

**Review by llxia — APPROVED** [review]
LGTM

**Pull request merged**